.github/workflows/build.yml

```
@@ -0,0 +1,101 @@
+name: Build
+
+on:
+  push:
+    paths-ignore:
+    - 'docs/**'
+  pull_request:
+    paths-ignore:
+    - 'docs/**'
+  schedule:
+    - cron: '0 8 * * *'
+
+jobs:
+
+  formatter:
+    runs-on: ubuntu-latest
+    name: Check java format
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 1
+    - uses: m273d15/java-format-action@v1.1
+      with:
+        version: '1.6'
+
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    # Pre-Build
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 1
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+
+    # Uncomment <depends> entries in order to make Saros/I installable in all JetBrains IDEs
+    - name: Prepare Saros/I plugin.xml
+      run: |
+        sed -i 's/<!--depends/<depends/g' intellij/resources/META-INF/plugin.xml
+        sed -i 's/depends-->/depends>/g' intellij/resources/META-INF/plugin.xml
+
+    # Build
+    - name: Build and test with Gradle
+      run: |
+        ./gradlew \
+          -PskipTestSuites=true \
+          -PaggregateTestResults=true \
+          -PversionQualifier=$GITHUB_SHA \
```

How about including a "DEV" (or "NIGHTLY", but I think I would prefer "DEV" in this situation; a matter of taste I guess) suffix to make it more clear that the build is a developer build. In that context, the appended SHA will also make more sense.

Also, as mentioned before, is there a way of sensibly shortening the SHA? Otherwise, the version string will be absurdly long.

```suggestion
          -PversionQualifier=DEV-$GITHUB_SHA \
```

---

> Also, as mentioned before, is there a way of sensibly shortening the SHA? Otherwise, the version string will be absurdly long.

I could try `git rev-parse --short <sha>`.

> How about including a "DEV".

Sounds reasonable

---

> I could try `git rev-parse --short <sha>`.

Sounds good.